querybook/webapp/components/UserBadge/UserBadge.tsx

```diff
@@ -89,4 +90,26 @@ export const UserBadge: React.FunctionComponent<IProps> = ({
             </div>
         </div>
     );
+
+    return groupPopover && userInfo?.is_group ? (
```

Comment:
show groupPopover always? when would you not show it?

Reply:
Just not sure if it's appropriate to display the popup for every use cases. But I can make it default to True.